src/lang/base.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
Do we even need a base lang file? Wouldn't it default to the input if the key is not found? Just asking, I don't know anything about localization in js :).

Reply:
I add this file to facilitate translation in other languages.